src/plays/tube2tunes/Tube2Tunes.js

```diff
@@ -154,4 +154,4 @@ function Tube2tunes(props) {
   );
 }
 
-export default Tube2tunes;
+export default Tube2Tunes;
```

Comment:
You don't need this anymore. Please revert.

Reply:
@atapas - reverted renaming changes.